src/pages/PostIndex.js

```diff
@@ -3,41 +3,42 @@ import { Card, CardBody, CardTitle, CardSubtitle, CardImg, Button } from "reacts
 import { NavLink } from "react-router-dom"
 
 function PostIndex ({ posts }) {
-  return (
-      <>
-      <main> 
-        {posts?.map((post, index) => {
-            <h2>Builds</h2>
-            return(
-                <div className="index-container" >
-                    <Card className="index-card" color='warning'
-                        style={{
-                            width: '18rem'
-                        }}
-                        key={index}
-                        >
-                    <CardImg className="index-image"
-                    alt={`profile picture for ${post.title}`}
-                    src={post.image}
-                    />
-                        <CardBody>
-                            <CardTitle tag="h5">
-                                {`Title: ${post.title}`}
-                            </CardTitle>
-                            <CardSubtitle tag="h6">
-                                {`Made By: ${post.userName}`}
-                            </CardSubtitle>
-                            <Button>
-                                <NavLink to={`/postsshow/${post.id}`} className="nav-link">
-                                See more details
-                                </NavLink>
-                            </Button>
-                        </CardBody>
-                    </Card>
-                </div>
-          )
-        })}
-      </main>
+    return (
+    <>
+        <main> 
+            {posts?.map((post, index) => {
+                <h2>Builds</h2>
+                return(
+                    
+                    <div className="index-container" >
+                        <Card className="index-card" color='warning'
+                            style={{
+                                width: '18rem'
+                            }}
+                            key={index}
+                            >
+                        <CardImg className="index-image"
+                        alt={`profile picture for ${post.title}`}
+                        src={post.image} 
+                        />
+                            <CardBody>
+                                <CardTitle tag="h5">
+                                    {`Title: ${post.title}`}
+                                </CardTitle>
+                                <CardSubtitle tag="h6">
+                                    {`Made By: ${post.userName}`}
+                                </CardSubtitle>
+                                <Button>
+                                    <NavLink to={`/postsshow/${post.id}`} className="nav-link">
+                                    See more details
+                                    </NavLink>
+                                </Button>
+                            </CardBody>
+                        </Card>
+                    </div>
+                )
+            })}
+        </main>
     </>
```

Comment:
just need to indent all of this to the right so everything is nested within the ()

Reply:
i cant believe i missed that, thank you